src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+}
+
+.red-block {
+  background-color: red;
+  height: 100px;
```

Comment:
you use `height: 100px;` three times
better to create class `block` and add this class to each `div` on the page

Example:
```
block {
  height: 100px;
}

.red-block {
  background-color: red;
}

...
```